composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "ezsystems/privacy-cookie-bundle",
+    "description": "Privacy cookie banner integration bundle into eZ Publish/eZ Platform",
+    "keywords": ["cookies", "privacy", "policy"],
+    "license": "GPL-2.0",
+    "authors": [
+        {
+            "name": "Rafal Toborek",
```

Comment:
Not a problem. My inspiration was: https://github.com/ezsystems/PlatformUIAssetsBundle/blob/master/composer.json#L7